packages/angular/common/src/providers/toast/toast.service.ts

```diff
@@ -27,8 +27,35 @@ export class ToastService {
     return getToastContainer().position;
   }
 
+  private createViewFromTemplate(
+    template: TemplateRef<any> | string | undefined,
+    context: any
+  ) {
+    if (template === undefined) {
+      return {
+        node: '',
+        embeddedView: undefined,
+      };
+    }
+    if (!(template instanceof TemplateRef)) {
+      return {
+        node: template as string,
+        embeddedView: undefined,
+      };
+    }
+
+    const embeddedView = template.createEmbeddedView({ $implicit: context });
+    const node = embeddedView.rootNodes[0];
+    embeddedView.detectChanges();
+
+    return { node, embeddedView };
+  }
+
   public async show(config: ToastConfig) {
-    if (typeof config.message === 'string') {
+    if (
+      typeof config.message === 'string' &&
+      !(config.action instanceof TemplateRef)
```

Comment:
This should also work, right?
```suggestion
      typeof config.message === 'string' && !config.action
```

Reply:
Yes, updated